deploy/roles/postgresql-server/tasks/main.yml

```diff
@@ -80,6 +85,7 @@
     owner: "{{ pp_serviceaccount }}"
     group: "{{ pp_serviceaccount }}"
     mode: 0700
+  when: ansible_distribution_major_version == "hell_froze_over" # TODO: figure out correct check
```

Comment:
What's the issue here?

Reply:
That was for developing; I've removed the "when" clause now.